internal/modules/circle/circle_report/dto/dto.go

```diff
@@ -0,0 +1,5 @@
+package circle_report_dto
+
+type CreateCircleReportBody struct {
```

Comment:
```suggestion
type CreateCircleReportPayload struct {
```
aku baru ganti name convention biar uniform semua, boleh tolong di ganti?

Reply:
noted